ci/packaging/suse/skuba_spec_template

```diff
@@ -32,7 +32,7 @@ Source1:        vendor.tar.gz
 Patch0:         0001-Patch-out-load-balancer.patch
 BuildRequires:  go-go-md2man
 BuildRequires:  golang-packaging
-BuildRequires:  golang(API) >= 1.12
+BuildRequires:  golang(API) = 1.13
```

Comment:
not `>= 1.13`?

Reply:
No, if we use `>= 1.13` then OBS gets confused about whether to use 1.13 or 1.14

```
$ osc build SLE_15_SP1 --no-verify
Building skuba.spec for SLE_15_SP1/x86_64
Getting buildconfig from server and store to /home/colleen/projects/packages/Devel:CaaSP:4.0/skuba/.osc/_buildconfig-SLE_15_SP1-x86_64
Getting buildinfo from server and store to /home/colleen/projects/packages/Devel:CaaSP:4.0/skuba/.osc/_buildinfo-SLE_15_SP1-x86_64.xml
buildinfo is broken... it says:
unresolvable: have choice for golang(API) >= 1.13: go1.13 go1.14
```

Reply:
If OBS have `go 1.13.8` and `golang(API) = 1.13`, OBS choose to use `1.13.0` or `1.13.8`?

Reply:
It chooses the only available go1.13 package, whether that's go 1.13.0 or 1.13.8, because the go1.13 package "provides" golang(API) 1.13, see line 170 of the [spec file](https://build.opensuse.org/package/view_file/devel:languages:go/go1.13/go1.13.spec?expand=1):

```
Provides:       golang(API) = %{go_api}
```